cliquenet/src/lib.rs

```diff
@@ -258,6 +276,9 @@ impl Network {
             peers.insert(k, a.into());
         }
 
+        let (otx, orx) = mpsc::channel(PEER_CAPACITY * peers.len());
```

Comment:
What's the logic behind this calculation? Is there any big difference between this and 10k? I recognize that it's a scaled factor of the peer size, but at the deployment sizes we expect, the computed value is nearly always << 10,000. Just curious.